tests/unit/Rules/CnpjTest.php

```diff
@@ -99,12 +99,13 @@ public function providerInvalidFormattedCnpj()
     public function providerInvalidUnformattedCnpj()
     {
         return [
-            ['11111111111'],
-            ['22222222222'],
-            ['12345678900'],
-            ['99299929384'],
-            ['84434895894'],
-            ['44242340000'],
```

Comment:
Those number are still invalid you shouldn't not remove then, just add the new ones.